_integration-schemas/stripe/v1/invoice_line_items.md

```diff
@@ -51,10 +51,9 @@ attributes:
     type: "boolean"
     description: "Indicates whether discounts can apply to this line item."
 
-  - name: "invoice"
+  - name: "id"
     type: "string"
-    description: "The ID of the invoice that contains this line item."
-    foreign-key-id: "invoice-id"
+    description: "The invoice line item ID."
```

Comment:
This is the other half of the new primary key for this table, so it should have a `primary-key: true` flag. Take look [here at the tap code](https://github.com/singer-io/tap-stripe/blob/543aaf799dbe26445bc74be9b0748a7a97e10099/tap_stripe/__init__.py#L27):

```
    'invoice_line_items': {'sdk_object': stripe.InvoiceLineItem,
                           'key_properties': ['id', 'invoice']}
```